src/Controller.js

```diff
@@ -166,7 +174,7 @@ class Controller {
 			return this.api.getData( window.location.href )
 				.then(
 					// Success handler.
-					() => {
+					result => {
```

Comment:
This patch fails for me when you launch, close-before-load, then launch again before the first load has finished (in the background). This is because, now the same promise is being returned from `Api.getData()`, there are two `.then()` invocations being branched on the same promise (i.e. not chained). The order in which they execute isn't determined, but the first one will do what we want, and the second one will break things. This is why, if you launch, close, then relaunch after 10 seconds or something, everything will work ok.

This means in practice that the 2nd invocation returns the tokenized HTML, but without the listeners, and so hovering is broken.

A (possibly naive) fix is to check at this point what state we're in, and not return the new HTML if we're already `ready`:

```javascript
if ( this.model.getState() === 'ready' ) {
	return;
}
```

This works in my testing.

Reply:
Ha! GOOD CATCH!!

I've added it to the condition that also checks if the model is active. Good one!